tests/contenttypes_tests/test_fields.py

```diff
@@ -33,3 +33,13 @@ def test_value_to_string(self):
         answer2 = Answer.objects.create(question=question)
         result = json.loads(Question.answer_set.field.value_to_string(question))
         self.assertCountEqual(result, [answer1.pk, answer2.pk])
+
+    def test_generic_reverse_relation_with_mti(self):
+        """
+        Checks that we still get the expected result using the reverse relation
```

Comment:
Docstrings should state the expected behavior and omit prefixes like "Checks that" since all tests check things.